myBlog/views.py

```diff
@@ -8,14 +8,58 @@
 from .serializers import PostSerializer
 from rest_framework.parsers import JSONParser
 from rest_framework import status
-from django.contrib.auth import login, authenticate
-from django.contrib.auth.forms import UserCreationForm
+from django.contrib.auth import login, authenticate, logout
+from django.contrib.auth.forms import UserCreationForm, AuthenticationForm
 from django.shortcuts import redirect
 from django.contrib.auth.models import User
 from django.views import generic
 from django.http import HttpResponse, JsonResponse
+from django.views.generic.edit import FormView
 
 
+class SignupView(FormView):
+    template_name = 'signup.html'
+    form_class = UserCreationForm  # The Form class the FormView should use
+    success_url = '/'  # Go here after successful POST
+
+    def form_valid(self, form):
+        username = form.cleaned_data.get('username')
+        password = form.cleaned_data.get('password1')
+
+        # is_active should equal true so user can login
+        User.objects.create_user(username=username, password=password, is_active=True)
```

Comment:
change "is_active=True" to "is_active=False", as one of the US 
As a server admin, I want to be able allow users to sign up but require my OK to finally be on my server